python3/httplib2/__init__.py

```diff
@@ -50,7 +49,30 @@
 from gettext import gettext as _
 import socket
 import ssl
-_ssl_wrap_socket = ssl.wrap_socket
+
+
+def ssl_wrap_socket(sock, key_file, cert_file, disable_validation,
+                     ca_certs, ssl_version, hostname):
+    if disable_validation:
+        cert_reqs = ssl.CERT_NONE
+    else:
+        cert_reqs = ssl.CERT_REQUIRED
+    if ssl_version is None:
+        ssl_version = ssl.PROTOCOL_TLS
+
+    if hasattr(ssl, 'SSLContext'):  # Python 3.2+
```

Comment:
Please put something like `if not SSLContext -> raise RuntimeError("Python 3.2+ required with ssl.SSLContext")`

Upon 2/3 merge we will not support 2.6- and 3.2- so SSLContext is automatically fulfilled.